File: python/encrypt.py
import base64
import hashlib
import json
import os
import sys
from Crypto.Cipher import AES

def encrypt(data, key):
    key = key.encode('utf-8')
    key = hashlib.sha256(key).digest()
    key = key[:16]
    data = data.encode('utf-8')
    cipher = AES.new(key, AES.MODE_ECB)
    encrypted = cipher.encrypt(pad(data))
    encrypted = base64.b64encode(encrypted)
    encrypted = encrypted.decode('utf-8')
    return encrypted

def pad(data):
    BS = 16
    return data + (BS - len(data) % BS) * chr(BS - len(data) % BS).encode('utf-8')

if __name__ == '__main__':
    key = 'A1B2C3D4E5F6H7I8'
    with open('../json.txt') as f:
        data = f.read()
        encrypted = encrypt(data, key)
        print("Encrypted: " + encrypted)

# Run
# python encrypt.py

File: rust/encrypt.rs
extern crate crypto;

use std::fs::File;
use std::io::prelude::*;
use crypto::symmetriccipher::SymmetricCipherError;
use crypto::aes::ecb_encryptor;
use crypto::buffer::{RefReadBuffer, RefWriteBuffer, BufferResult};
use crypto::buffer::ReadBuffer;
use crypto::buffer::WriteBuffer;
use crypto::blockmodes::PkcsPadding;
use crypto::util::FixedBuffer;

fn main() {
    let key = "A1B2C3D4E5F6H7I8";
    let mut file = File::open("../json.txt").expect("File not found");
    let mut data = String::new();
    file.read_to_string(&mut data).expect("Unable to read file");
    let encrypted = encrypt(&data, key);
    println!("Encrypted: {}", encrypted);
}

fn encrypt(data: &str, key: &str) -> String {
    let mut encryptor = ecb_encryptor(
        crypto::aes::KeySize::KeySize128,
        key.as_bytes(),
        PkcsPadding
    );
    let mut final_result = Vec::<u8>::new();
    let mut read_buffer = RefReadBuffer::new(data.as_bytes());
    let mut buffer = [0; 4096];
    let mut write_buffer = RefWriteBuffer::new(&mut buffer);
    loop {
        let result = encryptor.encrypt(&mut read_buffer, &mut write_buffer, true).unwrap();
        final_result.extend(write_buffer.take_read_buffer().take_remaining().iter().map(|&i| i));
        match result {
            BufferResult::BufferUnderflow => break,
            BufferResult::BufferOverflow => {}
        }
    }
    base64::encode(&final_result)
}

// Run
// rustc encrypt.rs